### ISSUE TITLE
Upgrade Ubuntu related images to 20.04

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#893]
We have a few Docker images that are still based on Ubuntu-16.04.  These we should add additional versions based on 20.04:

- [x] **ubuntu-16.04** (base)
- [x] **ubuntu-16.04-aspnet**
- [x] **ubuntu-16.04-dotnet**
- [x] **temporal-dev**? (we can probably delete this one)

**Comment by jefflill** [comment]
I went ahead and removed the **temporal-dev** image since we switched to using the DockerCompose and the stock Temporal.io images.

**Comment by jefflill** [comment]
It doesn't appear that we're actually referencing the Ubuntu-16.04 images any more since we added the **aspnet** and **dotnet** images sometime back.  Back in the the neonHIVE days we used **ubuntu-16.04** to host things like HashiCorp Vault, and HAPROXY but we no longer need to do that for Kubernetes.

I'm going to remove the build scripts but leave the images up on DockerHub for posterity.  If need a Linux base image in the future, we can try to base them off of **Alpine** or create more modern images.
